project-service/src/routes/projects.js

```diff
@@ -0,0 +1,149 @@
+import express from 'express';
+import models from '../models';
+import R from 'ramda';
+
+// for URLs
+// /projects/
+// /projects/:projectId
+
+let router = express.Router();
+
+let msg = {
+  user: { // user level (/projects)
+    400: {error: 'Invalid data parameters.'},
+    404: {error: "Project doesn't exist."},
```

Comment:
404 message isn't a valid response from GET or POST /projects

Can we rename user to be more descriptive? how about projects and project instead of user and project?